documentation/src/docs/asciidoc/release-notes-5.0.0-M5.adoc

```diff
@@ -54,4 +54,5 @@ on GitHub.
 
 ===== Bug Fixes
 
-* ❓
+* The `IsPotentialJUnit4TestClass` predicate incorrectly filtered out static nested classes.
+  It now let's them through properly
```

Comment:
Would this be better?

>Fixed a bug where static member classes were incorrectly being filtered out.
  They are now considered valid JUnit Vintage test classes.